generators/cypress/templates/cypress.json.ejs

```diff
@@ -25,6 +25,8 @@
   "fixturesFolder": "<%= CLIENT_TEST_SRC_DIR %>cypress/fixtures",
   "pluginsFile": "<%= CLIENT_TEST_SRC_DIR %>cypress/plugins/index.ts",
   "screenshotsFolder": "<%= BUILD_DIR %>cypress/screenshots",
+  "downloadsFolder": "<%= BUILD_DIR %>cypress/downloads",
+  "videosFolder": "<%= BUILD_DIR %>cypress/screenshots",
```

Comment:
Should this be videos instead of screenshots?

Reply:
Sure
```suggestion
  "videosFolder": "<%= BUILD_DIR %>cypress/videos",
```